src/map/unit.c

```diff
@@ -959,71 +964,60 @@ static enum unit_dir unit_getdir(const struct block_list *bl)
 	return ud->dir;
 }
 
-// Pushes a unit by given amount of cells into given direction. Only
-// map cell restrictions are respected.
-// flag:
-//  &1  Do not send position update packets.
-static int unit_blown(struct block_list *bl, int dx, int dy, int count, int flag)
+/**
+ * Pushes a unit by given amount of cells into given direction.
+ * @remark Only map cell restrictions are respected.
+ * @param bl: unit to push
+ * @param dir: direction of the push motion
+ * @param count: amount of cells to push
+ * @param no_update: don't send position update packets when true
+ * @return amount of pushed cells.
+ */
+static int unit_blown(struct block_list *bl, enum unit_dir dir, int count, bool no_update)
```

Comment:
Don't use negative names for booleans, That's bad practice.
Just have a look at this one:
`if (!no_update)`

This is read as "If not no update do ..." which is a double negative and doesn't make much sense.

Reply:
Will change to update instead

Reply:
I might have to rename the function in that case to have custom code not compile instead of having unexpected behavior.